app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -0,0 +1,73 @@
+//! Side menu for Welcome Screen.
+//!
+//! Side menu contains a list of available projects and "new project" button.
+
+use ensogl::prelude::*;
+
+use crate::ClickClosure;
+use crate::Frp;
+
+use ensogl::system::web;
+use ensogl::system::web::NodeInserter;
+use web_sys::Element;
+
+
+
+// ================
+// === SideMenu ===
+// ================
+
+/// Side menu for Welcome Screen. Contains a list of available projects and "new project" button.
+#[derive(Clone, CloneRef, Debug)]
+pub struct SideMenu {
+    logger:             Logger,
+    /// Root DOM element of the side menu.
+    pub root:           Element,
```

Comment:
Why is this comment needed? this is `root` inside of `SideMenu`. If you feel that such a comment is needed, it means that your variables are poorly named. Rename `root` to `root_dom` and remove the comment.

Reply:
It's quite interesting, lint is actually not needed (probably because the struct is technically crate-private). Renamed to `root_dom`.